src/sql/connection.py

```diff
@@ -50,25 +50,24 @@ def __init__(self, connect_str=None, connect_args={}, creator=None):
             raise
         self.dialect = engine.url.get_dialect()
         self.metadata = sqlalchemy.MetaData(bind=engine)
```

Comment:
Should passed-in names be proven to start with '@' or is this handled elsewhere (or implied)?